roles/eda/templates/redirect-page.configmap.html.j2

```diff
@@ -14,11 +14,11 @@ data:
         <title>Redirecting to Ansible Automation Platform</title>
 
         <!-- Favicon links -->
-        <link rel="icon" type="image/x-icon" href="static/rest_framework/docs/img/favicon.ico">
+        <link rel="icon" type="image/x-icon" href="/api/eda//static/rest_framework/docs/img/favicon.ico">
```

Comment:
```suggestion
        <link rel="icon" type="image/x-icon" href="/api/eda/static/rest_framework/docs/img/favicon.ico">
```
Is the // here correct? Or was this unintentional?

Reply:
Thanks for the review!  That was a bad replace, it is fixed now.  I also changed the favicon path to the new path.  I'll DM you with the other PR for that change in the container.